src/urllib3/poolmanager.py

```diff
@@ -471,7 +471,7 @@ def urlopen(  # type: ignore[override]
             kw["headers"] = new_headers
 
         try:
-            retries = retries.increment(method, url, response=response, _pool=conn)  # type: ignore[arg-type]
+            retries = retries.increment(method, url, response=response, _pool=conn)
```

Comment:
I guess `increment()` took a `HTTPResponse` from `urllib3` and now it's finally getting one? All I know is `mypy` complained about it.